### PR TITLE
Enable support for s390x in conda-build

### PR DESCRIPTION
**Pull request by pradghos**
Addressing : https://github.com/conda/conda/issues/9791
Enable support for `s390x` in `conda-build`.

### CODE REVIEW

**Comment by jjhelmus** [comment]
Looks good to me.  @pradghos would it be possible for you to sign the Contributor License Agreement?  If you already have let me know and I'll take a look.

File: conda_build/convert.py [hunk 1]
```diff
@@ -763,7 +764,7 @@ def conda_convert(file_path, output_dir=".", show_imports=False, platforms=None,
     source_platform_architecture = '{}-{}' .format(source_platform, architecture)
 
     if 'all' in platforms:
-        platforms = ['osx-64', 'linux-32', 'linux-64', 'linux-ppc64',
+        platforms = ['osx-64', 'linux-32', 'linux-64', 'linux-ppc64','linux-s390x',
```
Comment (hartb) [inline conda_build/convert.py:767]: Minor, but might be nice to swap the order of `ppc64le` and `s390x` here to keep the two `ppc*` arch's together in the list.

**Comment by pradghos** [comment]
was on vacation .. I will come back to this PR soon ! Thanks!

**Comment by pradghos** [comment]
> 
> 
> Looks good to me. @pradghos would it be possible for you to sign the Contributor License Agreement? If you already have let me know and I'll take a look.

@jjhelmus Sorry for the delayed response ! I have signed Contributor License Agreement , Could you please have a look ? Thank you !

File: conda_build/convert.py (hunk 1 above)
Comment (pradghos) [reply conda_build/convert.py]: @hartb : I have addressed this comment in the latest commit. Thanks!

**Comment by pradghos** [comment]
Please let me know in case of any review comments to address further. Thank you very much !

**Comment by jjhelmus** [comment]
@cla-bot check

**Comment by jjhelmus** [comment]
LGTM, thanks @pradghos

**Pull request merged**